### DISCUSSION TITLE
Poll: Weigh in on possible new sections to add

### DISCUSSION THREAD

**Discussion by maryjom** [w3c/wcag2ict#48] [Polls]
I'm going to try doing this in a poll, just to try out this feature. If you wish there was a different choice available, or have some commentary, add it in a comment at the bottom of the poll. 

For W3C standards that get updated, there are sections describing changes between versions, background/history, etc.  Working Group Notes (like WCAG2ICT) typically don't have them. However, since this may become more of a formalized document, I would like your input on whether we should have any (or all) of the sections listed in the poll below in the updated WCAG2ICT note.  

As an example, WCAG 2.2 has these sections:
- [Background on WCAG 2](https://www.w3.org/TR/WCAG22/#background-on-wcag-2)
- [Comparison with WCAG 2.1](https://www.w3.org/TR/WCAG22/#comparison-with-wcag-2-1)
- [Change log](https://www.w3.org/TR/WCAG22/#changelog)

**Comment by bruce-usab** [comment]
Polls are one more new-to-me GitHub feature!  :heart:

I think a narrative comparison to WCAG2ICT would be preferable to something like a "change log".  WCAG 2.1 and 2.2 include lists of new SC (something which I think is quite helpful), but I am undecided if that approach will work well for WCAG2ICT.

**Reply by maryjom** [reply]
@bruce-usab I tend to agree, WCAG2ICT wouldn't need a super-detailed listing of changes like a technical standard, but a more general description of what is new/changed, like: 

- Added 12 new SCs and a guideline from WCAG 2.1 with a link to the WCAG 2.1 new features list
- Added 8 new SCs from WCAG 2.2 with a link to the WCAG 2.2 new features list
- Updated information about applying WCAG to software with closed functionality (or identifying how we've changed things for this topic)

There is also the possibility of having some way of marking each of the SC with what version(s) of WCAG they appeared in.

**Reply by mapluke** [reply]
That sounds about right to me.

**Comment by Lboniello** [comment]
A comparison chart might be useful with explanation/definition (more narrative) for each line in the chart as needed (expand for more narrative description upon click)

**Comment by GreggVan** [comment]
relied upon in regulations is a good way to look at it

describe history of regulatory use since 1st version

**Reply by maryjom** [reply]
Thought I'd add here the particular mentions of the previous WCAG2ICT Note here:
* [508 - Application of WCAG 2.0 to non-web ICT](https://www.access-board.gov/ict/#b-application-of-wcag-2-0-to-non-web-ict)
* [EN 301 549 V3.2.1 - Informative references](https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf#page=[13]) number [i.26].
* and any standards that incorporate the EN 301 549 (e.g. India, Kenya, Australia, etc.)

**Comment by bruce-usab** [comment]
Access Board also has:
- [Comparison Table of WCAG 2.0 to Original 508 Standards](https://www.access-board.gov/ict/wcag2ict.html)
- [Map of WCAG to Section 508 FPC](https://www.access-board.gov/ict/wcagtofpc.html)

There is a similar, but not exactly the same, FPC to SC mapping in EN 301 549.  Neither of the above two resources are specific to non-web documents and software, but they may be helpful to some of our analysis.

**Reply by maryjom** [reply]
True, but these are not necessarily WCAG2ICT Note related, right? The WCAG2ICT work doesn't include any mappings to FPC (in 508) or FPS (in the EN).

**Comment by maryjom** [comment]
This was resolved and implemented with a new Background section and a Comparison section.
